CMakeLists.txt

```diff
@@ -143,6 +143,7 @@ if(BUILD_WASM)
     add_link_options(-pthread)
     add_link_options(-sPTHREAD_POOL_SIZE=8)
     add_compile_options(-s DISABLE_EXCEPTION_CATCHING=0)
+    add_link_options(-sSTACK_SIZE=8388608) # 8MB
```

Comment:
@mewim what do you think is a good number for this?

Reply:
Yeah I think this is fine. The default stack size in most Linux distro is 8MB. But I think `sSTACK_SIZE` does accept `-sSTACK_SIZE=8MB`. I think we should change bytes to MB for better readability and the comment can be removed.